### ISSUE TITLE
I see this error when starting "ananicy dump proc"

### ISSUE THREAD

**Issue by servimo** [Nefelim4ag/Ananicy#418]
I see this error when starting "ananicy dump proc"

File: /etc/ananicy.d/00-cgroups.cgroups, Line: 6, Error: cgroup fs: cpu missing
File: /etc/ananicy.d/00-cgroups.cgroups, Line: 7, Error: cgroup fs: cpu missing

I am getting high CPU when compiling C++

**Comment by Nefelim4ag** [comment]
You missing cpu cgroup in machine by some way, I think you use custom kernel which not support cpu cgroup

**Comment by servimo** [comment]
No, not using custom kernel, maybe some configuration I did. I am using the kernel 5.14 or 5.10 from Manjaro, both happens this. This "Error: cgroup fs: cpu missing" means ananicy is not working?

**Comment by servimo** [comment]
No, not using a custom kernel. I use normal kernels from Manjaro. Maybe is some configuration I did blind. In what other places I can find configs to the kernel cgroup? I see that there is a /sys/fs/cgroup in my system.

**Comment by servimo** [comment]
I delete in 00-types.types:
"cgroup": "cpu80" 
"cgroup": "cpu90"

and then added:
"CPUQuota": 80
"CPUQuota": 90

**Comment by servimo** [comment]
Adding this to /etc/default/grub line GRUB_CMDLINE_LINUX_DEFAULT  "systemd.unified_cgroup_hierarchy=0" and then doing a "sudo update-grub" brings back the cgroup.

**Comment by Nefelim4ag** [comment]
Hm... maybe I will take a look at cgroup2 when I have free time because currently I just don't use Ananicy at all
